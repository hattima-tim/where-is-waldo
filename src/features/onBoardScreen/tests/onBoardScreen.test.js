import { render } from "@testing-library/react";
import OnBoardScreen from "../onBoardScreen";

jest.mock('../character',()=>({characterName})=>{
    return <p>{characterName}</p>
});

jest.mock('../userInstructionCard',()=>()=>{
    return <p>User Instruction Card</p>
})

test('renders onBoardScreen correctly',()=>{
    const {container} = render(<OnBoardScreen/>);

    expect(container).toMatchSnapshot();
})