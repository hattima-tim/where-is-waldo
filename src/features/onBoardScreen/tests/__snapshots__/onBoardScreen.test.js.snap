// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders onBoardScreen correctly 1`] = `
<div>
  <div
    class="false z-10 flex w-full flex-col items-center justify-center gap-6 bg-[rgba(0,0,0,0.9)]"
  >
    <h1
      class="mt-8 text-lg font-bold text-white"
    >
      Where's Waldo
    </h1>
    <p
      class="text-white"
    >
      Find these characters on universe 113!
    </p>
    <p>
      User Instruction Card
    </p>
  </div>
</div>
`;
